### ISSUE TITLE
Incorrect K8s version is displayed on a cluster edit when changing from revision with v1.13 latest to v1.14 latest

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#23648]
Version v2.3-head

Steps

1. Create a cluster template T1 and Revision R1 using K8s version v1.13 [Latest] and no override set
2. Create revision R2 using K8s version v1.14(Latest) and no override set
3. Create a cluster using R1
4. Edit the cluster and choose R2.

The K8s version points to v1.13.2-rancher1-1 and when looking at the dropdown list, v1.14.8-rancher1-1 is greyed out. 

<img width="1669" alt="Screen Shot 2019-10-23 at 3 04 18 PM" src="https://user-images.githubusercontent.com/18536626/67438083-6a1ade00-f5a7-11e9-8c12-a6cea2453a74.png">

Only after refresh, v1.14.8-rancher1-1 is highlighted and is available to choose.

**Comment by deniseschannon** [comment]
Seems like a duplicate of https://github.com/rancher/rancher/issues/23616

**Comment by westlywright** [comment]
This is not a duplicate. The difference is the patch version to patch version upgrade. While the other is an exact to patch version upgrade scenario.

**Comment by westlywright** [comment]
Available in `latest-2.3`.

**Comment by soumyalj** [comment]
Tested with v2.3-head with UI index latest-2.3
Steps:

1.Create a cluster template T1 and Revision R1 using K8s version v1.13 [Latest] and no override set
2. Create revision R2 using K8s version v1.14(Latest) and no override set
3. Create a cluster using R1
4. Edit the cluster and choose R2.
The K8s version points to  v1.14.8-rancher1-1 as expected
<img width="661" alt="Screen Shot 2019-10-30 at 3 06 09 PM" src="https://user-images.githubusercontent.com/18536626/67902677-f89eda80-fb26-11e9-9e34-3ee00848004a.png">

Dropdown view
<img width="602" alt="Screen Shot 2019-10-30 at 3 06 01 PM" src="https://user-images.githubusercontent.com/18536626/67902680-fd638e80-fb26-11e9-8e49-b94d65e65312.png">
